sprite.go

```diff
@@ -492,6 +492,7 @@ func (p *Sprite) Die() {
 		p.goAnimate(aniName, ani)
 	}
 
+	p.Stop(OtherScriptsInSprite)
```

Comment:
I tested with a sample project [die.zip](https://github.com/user-attachments/files/16973112/die.zip)

Just like what I mentioned in #315

While it still ouputs like this:

```
im alive
im alive
im alive
im alive
im alive
im alive
im alive
im alive
im alive
im alive
im alive
click
im alive
im alive
im alive
im alive
im alive
im alive
im alive
```

That is not expected.